### ISSUE TITLE
The keyboard focus does not move on the working files button

### ISSUE THREAD

**Issue by bazzdg** [Microsoft/monaco-editor#969]
**Actual**: While navigating the menu item of the explore button via of the keyboard, the focus moves on the workspace and does not move on the working files button via of the keyboard. 

**Expected**: All the controls should be accessible via tab key of the keyboard. The keyboard focus should move on the working files button and should be expanded on pressing enter key on it. 

**Note**: In scan mode of the narrator the focus moves on the working files button. User Impact: Keyboard only user will find difficulty if the keyboard focus does not move on the working files button. 

**Test Environment:** 
- OS: Win 10 
- OS version: 1709
- OS build: 16299.98
- Browser : Edge 

**MAS Reference:** MAS 2.1.1- Keyboard.

![workingfilesmonaco](https://user-images.githubusercontent.com/38893529/43016457-c20646d2-8c07-11e8-85e4-1bbfd1e4b188.png)

**Comment by alexandrudima** [comment]
@bazzdg What product are you testing? That is outside of the monaco editor.
